CarProgram/src/carprogram/Car.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package carprogram;
+
+/**
+ *
+ * @author Zachary
+ */
+public class Car {
+    private String make= "Ford";
+    private int speed = 0;
+    private int yearModel = 2000;
```

Comment:
There is no need to provide defaults here at the field level they will be resolved by the constructor.
Specific the speed should be set to 0 there